_pages/understanding/upgrading.md

```diff
@@ -65,34 +59,11 @@ Use the following procedure to upgrade your project to a new version of DHF usin
 
    {% include ostabs.html linux="./gradlew hubUpdate" windows="gradlew.bat hubUpdate" %}
 
-The `hubUpdate` task might report incompatibilities that you must correct.
-
-You should also update the DHF version in the `dependencies` section of any custom build.gradle files that depend on DHF.
-
-From 4.0.0 on, the gradle properties file must indicate the version of the project.  This property is to tell DHF how to interpret
-the project structure so that we can support future upgrade paths more easily.  So as a last step, include
-
-mlDHFVersion=4.0.0
-
-
-### Upgrading Using QuickStart
-**NOTE**: Do not use QuickStart to upgrade DHF 3.0.0 to DHF 4.0.0 or DHF 4.0.1.
+  The `hubUpdate` task might report incompatibilities that you must correct.
 
-Use the following procedure to upgrade your project to a new version of DHF using QuickStart. You do not need to perform these steps if you already upgraded using `ml-gradle`.
+1. If you upgrade to DHF 4.0.0 or later, update your gradle properties file to set `mlDHFVersion` to your new DHF version. For example, if you upgrade your "local" environment to DHF 4.0.1, then your `local.properties` file should contain the following: `mlDHFVersion='4.0.1'`.
```

Comment:
this would be gradle-local.properties.
However I kind of thing this environment thing is a bad practice anyway, so you might consider sticking to gradle.properties.